packages/client/lib/util/parse.ts

```diff
@@ -166,7 +166,7 @@ async function parseGethParams(json: any) {
     genesis: {
       hash,
       timestamp,
-      gasLimit,
+      gasLimit: parseInt(gasLimit), // Geth stores gasLimit as a hex string while our gasLimit is a `number`
```

Comment:
since it's a hex string you'll want to specify radix of 16
```suggestion
      gasLimit: parseInt(gasLimit, 16), // Geth stores gasLimit as a hex string while our gasLimit is a `number`
```

Reply:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt), if it encounters 0x at the start of the string, it treats it as hexadecimal. From what I observe, geth always attaches the 0x at the front so this should not be an issue.  But, no strong opinion.